go/test/endtoend/vault/vault_test.go

```diff
@@ -267,6 +267,12 @@ func initializeClusterLate(t *testing.T) {
 		require.NoError(t, err)
 	}
 
+	for _, tablet := range shard.Vttablets {
+		// remove super read-only from vttablet
+		tablet.VttabletProcess.UnsetReadOnly("")
```

Comment:
As noted elsewhere, I think `tablet.VttabletProcess.SetReadOnly(false)` is nicer.